app/src/main/java/com/example/roomremix2/DrawView.kt

```diff
@@ -22,16 +22,16 @@ class DrawView : View {
     }
     //import constructor view
     constructor(context: Context) : this(context, null){
-        init()
+        screenSmooth()
     }
     constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, 0){
-        init()
+        screenSmooth()
     }
 
     constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {
-        init()
+        screenSmooth()
     }
-    private fun init(){
+    private fun screenSmooth(){
```

Comment:
init is the proper name for an initialization function, as this one appears to be.  I'd keep this as it were.

Reply:
This is a direct copy of another student's code review.